src/layout/tests.rs

```diff
@@ -31,6 +31,7 @@ struct TestWindowInner {
     is_fullscreen: Cell<bool>,
     is_windowed_fullscreen: Cell<bool>,
     is_pending_windowed_fullscreen: Cell<bool>,
+    is_urgent: Cell<bool>,
```

Comment:
If removing `set_urgent()`, then this is not necessary, can just return constant false from `is_urgent()`.